packages/core-data/src/queried-data/reducer.js

```diff
@@ -1,7 +1,7 @@
 /**
```

Comment:
I think we may also need to update the `entity.edits` reducer in `packages/core-data/src/reducer.js` to remove any stored edits when `DELETE_ITEMS` is dispatched.

Reply:
I think you are right.